themes/terrain-texture.yaml

```diff
@@ -35,17 +26,162 @@ layers:
+                DASH_SIZE: 0.9
+                DASH_TYPE: fill
+                DASH_TILE_STYLE: tile
+                PI: 3.141592653589793
+            blocks:
+                global: |
+                    float dashDF(vec2 st) {
+                        return min(cos((st.x-st.y)*PI),-cos((st.x+st.y+.5)*PI*.6666)*5.);
+                    }
+                color: |
+                    color.rgb = mix(DASH_BACKGROUND_COLOR, 
+                                    DASH_COLOR, 
+                                    DASH_TYPE( DASH_SIZE, dashDF(DASH_TILE_STYLE(getTileCoords()*DASH_SCALE,3.))) );
+    shapes-type:
+        mix:
+            - functions-aastep
+        shaders:
+            defines:
+                STROKE: 0.15
+            blocks:
+                global: |
+                    float fill (in float size, in float x) {
+                        return 1.-aastep(size, x);
+                    }
+
+                    float stroke (in float size, in float x) {
+                        return aastep(size, x+STROKE*.5) - aastep(size, x-STROKE*.5);
+                    }
+    elevation-normal:
+        raster: custom
+        shaders:
+            defines:
+                NORMAL_TEXTURE_INDEX: 0
+            blocks:
+                normal: |
+                    vec4 normal_elv_raster = sampleRaster(int(NORMAL_TEXTURE_INDEX));
+                    normal = (normal_elv_raster.rgb-.5)*2.;
+    polygons-dots:
+        base: polygons
+        mix:
+            - space-tile
+            - tiling-brick
+            - tiling-tile
+            - shapes-circle
+        shaders:
+            defines:
+                DOTS_COLOR: color.rgb*.5
+                DOTS_BACKGROUND_COLOR: color.rgb
+                DOTS_SCALE: 10
+                DOTS_SIZE: 0.41
+                DOTS_TYPE: fill
+                DOTS_TILE_STYLE: brick
+            blocks:
+                color: |
+                    color.rgb = mix(DOTS_BACKGROUND_COLOR, 
+                                    DOTS_COLOR, 
+                                    DOTS_TYPE( DOTS_SIZE, circleDF(vec2(0.5)-DOTS_TILE_STYLE(getTileCoords()*DOTS_SCALE,2.))) );
+    shapes-circle:
+        mix:
+            - shapes-type
+        shaders:
+            blocks:
+                global: |
+                    // get distance field of a Circle
+                    float circleDF (vec2 st) {
+                                    return dot(st,st)*3.03;
+                    }
+
+                    // Draw a circle in the middle of the ST space
+                    float circle (vec2 st, float radius) {
+                                    return fill(radius, circleDF(st-vec2(0.5)));
+                    }
+
+                    // Draw a circle in the middle of the ST space
+                    float circleBorder (vec2 st, float radius) {
+                                    return stroke(radius, circleDF(st-vec2(0.5)));
+                    }
+    elevation-ramp:
+        mix: colorized_terrain
```

Comment:
please always use the `-` or `[]` instead of some of each.

Reply:
(You're mostly using `-`, but then @sensescape is using `[]`, please standardize that in this PR, probably to `-`.)

Reply:
addressed with 68f2129a590b3290752febfff9fc33cb7d056fb3

Reply:
Oops, just saw this – the `-` came from the yaml-merging tool I used to squish all the imports together, I tend toward `[]` for brevity but legibility is good too, I'll switch them back over.